CalDavSynchronizer/Globalization/StringResources.ru-RU.resx

```diff
@@ -612,7 +612,7 @@
   <data name="Network Settings" xml:space="preserve">
     <value>Сетевые настройки</value>
   </data>
-  <data name="Password:" xml:space="preserve">
+  <data name="Hitask Account Passwort:" xml:space="preserve">
```

Comment:
Should be password, not "passwort", here is in english.

Reply:
update done